src/connectors/ldap.py

```diff
@@ -0,0 +1,219 @@
+"""LDAP Logs
+Collect LDAP logs from S3 using AssumeRole
+"""
+
+from json import dumps
+from time import sleep
+import yaml
+
+from runners.helpers import db
+
+from .utils import yaml_dump
+
+CONNECTION_OPTIONS = [
+    {
+        'type': 'str',
+        'name': 'bucket_name',
+        'title': "LDAP Log Bucket",
+        'prompt': "The S3 bucket where the LDAP Logs are collected",
+        'prefix': "s3://",
+        'placeholder': "my-ldap-log-s3-bucket",
+    },
+    {
+        'type': 'str',
+        'name': 'prefix',
+        'title': "Prefix Filter",
+        'prompt': "The folder in LDAP Log Bucket where logs are collected",
+        'default': "ldap/",
```

Comment:
how common is default? is it something only we have?

Reply:
IIRC ldap logs are ingested to S3 via FluentD, so there's not really a meaningful default to put here; it's what we set up when we configured fluentd.